src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -142,11 +127,24 @@
       ringutils/wrap-response-logging
       (ringutils/wrap-with-puppet-version-header puppet-version)))
 
-(defn construct-invalid-request-handler
-  "Constructs a ring handler to handle an incorrectly formatted request and indicate to the user
-   they need to update to Puppet 4"
-  [error-message]
-  (fn
-    [_]
-    {:status 404
-     :body   error-message}))
+(schema/defn ^:always-validate get-master-route-config
```

Comment:
These functions were moved from the `master_service` namespace and altered a bit.  Thought it would be better for them to live here so that various users of these functions don't have to call into "service" layer code for a service that may not be loaded, e.g., when executed via the PE master service.